build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cargo clippy --all-targets --all-features -- -D warnings
+# NOTE: for --release builds also change ./run to actually use it!
```

Comment:
We could make the build and run script always build and use release versions and add additional scripts if we want build scripts for ourselves. If we have to modify these scripts every week before a release sounds like a bad idea IMO.

Reply:
If it's only used for speedcenter builds then yes.
On the other hand, I find it handy to have a local script to check whether CI will complain about a commit I make (before pushing).

Reply:
we could just make an additional script `run_debug`?

Reply:
Yes I can see this. We could add a `build-debug` script for this. This would allow to also skip some steps that are maybe necessary on a release, but not in development. Or the other way around.

E.g. for the speedcenter we don't need a Clippy check but _I_ want to enforce this here.